### PR TITLE
user invite functionality added

### PR DESCRIPTION
**Pull request by Anil-CM**

### CODE REVIEW

File: api/iam/iamv2/api_service.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	bluemix "github.com/IBM-Cloud/bluemix-go"
+	"github.com/IBM-Cloud/bluemix-go/authentication"
+	"github.com/IBM-Cloud/bluemix-go/client"
+	"github.com/IBM-Cloud/bluemix-go/http"
+	"github.com/IBM-Cloud/bluemix-go/rest"
+	"github.com/IBM-Cloud/bluemix-go/session"
+)
+
+//IAMServiceAPI is the resource client ...
+type IAMServiceAPI interface {
```
Comment (hkantare) [inline api/iam/iamv2/api_service.go]: change IAMServiceAPI to UserManagementAPI

File: api/iam/iamv2/api_service.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	UserInvite() Users
+}
+
+//ErrCodeAPICreation ...
+const ErrCodeAPICreation = "APICreationError"
+
+//iamService holds the client
+type iamService struct {
```
Comment (hkantare) [inline api/iam/iamv2/api_service.go]: Same as above

File: examples/iam/user_invite/main.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+func main() {
+
+	const (
+		Member = "MEMBER"
+	)
+	var userEmail string
+	flag.StringVar(&userEmail, "userEmail", "", "Email of the user to be invited")
+
+	var accountID string
```
Comment (hkantare) [inline examples/iam/user_invite/main.go]: Is it IBMId or accountID?

Comment (Anil-CM) [reply examples/iam/user_invite/main.go]: @hkantare accountID of the inviter is used.

**Pull request merged**